solutions/fully-configurable/variables.tf

```diff
@@ -28,7 +28,19 @@ variable "existing_resource_group_name" {
 variable "prefix" {
   type        = string
   nullable    = true
-  description = "The prefix to add to all resources that this solution creates (e.g `prod`, `test`, `dev`). To not use any prefix value, you can set this value to `null` or an empty string.`"
+  description = "The prefix to be added to all resources created by this solution. To skip using a prefix, set this value to null or an empty string. The prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It should not exceed 16 characters, must not end with a hyphen('-'), and can not contain consecutive hyphens ('--'). Example: prod-0205-cos.[Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/issue_13328/solutions/fully-configurable/DA-prefix.md)"
```

Comment:
[Learn more] link change to https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines.md